scala-thoughts.md

```diff
@@ -0,0 +1,55 @@
+## Scala thoughts
+
+#### Likes
+
+Overall, my impression of Scala is that it is very intuitive.
+To me, it seems most similar to Python, but with static typing.
+
+I personally like the static typing because it can make the code a lot more
+readable and make it easier to understand what the input and output of each
+method is.
```

Comment:
I agree with you here! I find that static typing is helpful when I'm writing code nowadays, as it lets me denote exactly what that datatype should be so I can know what's going on at a glance. But I can remember back to when I was first learning to code, and I definitely preferred dynamic typing back then so I could focus on more significant parts of the code. Maybe languages should try and have both?